src/services/ProfileService.ts

```diff
@@ -39,7 +39,7 @@ const useHydrateProfiles = () => {
     }, [accessToken, userId, dispatch]);
 };
 
-const useSetFavoriteCommander = () => {
+const useSetServerProfile = () => {
```

Comment:
nit: probably useSetProfile is better. or even useUpdateProfile

server doesn't add very much here since the profile is always coming from the service

Reply:
The reason why I used "set" before was that it was a specific field. When I am referring to the entire profile, it is probably just an "update"